expyfun/io/_parse.py

```diff
@@ -73,3 +99,117 @@ def read_tab(fname, group_start='trial_id', group_end='trial_ok'):
             d[key] = [(these_vals[ii], these_times[ii]) for ii in idx]
         data.append(d)
     return data
+
+
+def reconstruct_tracker(fname):
+    """Reconstruct TrackerUD and TrackerBinom objects from .tab files.
+
+    Parameters
+    ----------
+    fname : str
+        Input filename.
+
+    Returns
+    -------
+    tr : list of TrackerUD or TrackerBinom
+        The tracker objects with all responses such that they are in their
+        stopped state (as long as the trackers were allowed to stop during
+        the generation of the file.) If only one tracker is found in the file,
+        it will still be stored in a list and will be assessible as ``tr[0]``.
+    """
+    from ..stimuli import TrackerUD, TrackerBinom
+    # read in raw data
+    raw = read_tab_raw(fname)
+
+    # find tracker_identify and make list of IDs
+    tracker_idx = np.where([r[1] == 'tracker_identify' for r in raw])[0]
+    if len(tracker_idx) == 0:
+        raise ValueError('There are no Trackers in this file.')
+    tr = []
+    for ii in tracker_idx:
+        tracker_id = ast.literal_eval(raw[ii][2])['tracker_id']
+        tracker_type = ast.literal_eval(raw[ii][2])['tracker_type']
+        # find tracker_ID_init lines and get dict
+        init_str = 'tracker_' + str(tracker_id) + '_init'
+        tracker_dict_idx = np.where([r[1] == init_str for r in raw])[0][0]
+        tracker_dict = json.loads(raw[tracker_dict_idx][2])
+        if tracker_type == 'TrackerUD':
```

Comment:
trying to understand this "trick"...  doesn't your suggested redefinition of `tracker_dict` overwrite the `json.loads` definition?  Did you mean to say something like 
```python
tracker_dict = json.loads(raw[tracker_dict_idx][2])  # unchanged
td = dict(TrackerUD=TrackerUD, TrackerBinom=TrackerBinom)  # changed var name
tr.append(td[tracker_type](**tracker_dict))
```